.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: required
 language: java
 jdk:
-- oraclejdk8
```

Comment:
Pull request naming convention to be followed while creating a PR

Reply:
Updated